src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -227,15 +237,15 @@ def test_WINNF_FT_S_PCR_1(self, config_filename):
     uut_ppa_geometry = ppa_zone_data[0]['zone']['features'][0]['geometry']
 
     # Configure the Census Tract Directory that the PPA uses
-    ppa.ConfigureCensusTractDriver(censustract_data_path)
+    ppa.ConfigureCensusTractDriver(DEFAULT_CENSUSTRACTS_DATAPATH)
 
     # PpaCreationModel requires the input registrationRequests to have 'installationParam'.
     # But this parameter is removed for devices where conditionals are pre-loaded.
     # Adding the 'installationParam' into registrationRequests by taking the
     # corresponding values from conditionalRegistrationData.
     for device in config['registrationRequests']:
```

Comment:
Will this make a copy of the dict or use a reference? If it's making a copy, the values won't be updated below.

Reply:
There is no implicit copy in python, everything is always a reference. Unless I misunderstood the question.

Reply:
Be   default object reference are passed unless if it is specified in python. No   changes is expected here and tested and works as expected.